node/typings/Checkout.ts

```diff
@@ -59,6 +64,7 @@ interface OrderFormItem {
   sellerChain: string[]
   availability: string
   unitMultiplier: number
+  assemblies: CheckoutAssemblyItem[]
```

Comment:
I think `assemblyItems` would be a better name

Reply:
isso é o tipo da resposta do checkout, não é eu que escolho nome, só estou tipando